cid-decode/src/multicodecs.rs

```diff
@@ -0,0 +1,617 @@
+            (0xb3ce, Multicodec::new("skein1024-880", "multihash")),
+            (0xb3cf, Multicodec::new("skein1024-888", "multihash")),
+            (0xb3d0, Multicodec::new("skein1024-896", "multihash")),
+            (0xb3d1, Multicodec::new("skein1024-904", "multihash")),
+            (0xb3d2, Multicodec::new("skein1024-912", "multihash")),
+            (0xb3d3, Multicodec::new("skein1024-920", "multihash")),
+            (0xb3d4, Multicodec::new("skein1024-928", "multihash")),
+            (0xb3d5, Multicodec::new("skein1024-936", "multihash")),
+            (0xb3d6, Multicodec::new("skein1024-944", "multihash")),
+            (0xb3d7, Multicodec::new("skein1024-952", "multihash")),
+            (0xb3d8, Multicodec::new("skein1024-960", "multihash")),
+            (0xb3d9, Multicodec::new("skein1024-968", "multihash")),
+            (0xb3da, Multicodec::new("skein1024-976", "multihash")),
+            (0xb3db, Multicodec::new("skein1024-984", "multihash")),
+            (0xb3dc, Multicodec::new("skein1024-992", "multihash")),
+            (0xb3dd, Multicodec::new("skein1024-1000", "multihash")),
+            (0xb3de, Multicodec::new("skein1024-1008", "multihash")),
+            (0xb3df, Multicodec::new("skein1024-1016", "multihash")),
+            (0xb3e0, Multicodec::new("skein1024-1024", "multihash")),
+            (
+                0xb401,
+                Multicodec::new("poseidon-bls12_381-a2-fc1", "multihash"),
+            ),
+            (
+                0xb402,
+                Multicodec::new("poseidon-bls12_381-a2-fc1-sc", "multihash"),
+            ),
+            (0xb501, Multicodec::new("ssz", "serialization")),
+            (0xb502, Multicodec::new("ssz-sha2-256-bmt", "multihash")),
+            (0xcc01, Multicodec::new("iscc", "softhash")),
+            (
+                0xce11,
+                Multicodec::new("zeroxcert-imprint-256", "zeroxcert"),
+            ),
+            (
+                0xf101,
+                Multicodec::new("fil-commitment-unsealed", "filecoin"),
+            ),
+            (0xf102, Multicodec::new("fil-commitment-sealed", "filecoin")),
+            (0x706c61, Multicodec::new("plaintextv2", "multiaddr")),
+            (0x807124, Multicodec::new("holochain-adr-v0", "holochain")),
+            (0x817124, Multicodec::new("holochain-adr-v1", "holochain")),
+            (0x947124, Multicodec::new("holochain-key-v0", "holochain")),
+            (0x957124, Multicodec::new("holochain-key-v1", "holochain")),
+            (0xa27124, Multicodec::new("holochain-sig-v0", "holochain")),
+            (0xa37124, Multicodec::new("holochain-sig-v1", "holochain")),
+            (0xb19910, Multicodec::new("skynet-ns", "namespace")),
+            (0xb29910, Multicodec::new("arweave-ns", "namespace")),
+            (0xb39910, Multicodec::new("subspace-ns", "namespace")),
+            (0xb49910, Multicodec::new("kumandra-ns", "namespace")),
+        ]);
+        m
```

Comment:
You can probably just `HashMap::from(...)` instead of assigning and returning it, I think.